mysql-test/suite/galera_sr/r/galera_sr_shutdown_master.result

```diff
@@ -26,4 +28,4 @@ COUNT(*) = 3
 1
 DROP TABLE t1;
 connection node_2;
-CALL mtr.add_suppression("WSREP: failed to send SR rollback for ");
+CALL mtr.add_suppression("WSREP: Failed to replicate rollback fragment for ");
```

Comment:
I did not get this warning.

Reply:
worker[1] Using MTR_BUILD_THREAD 300, with reserved ports 16000..16019
galera_sr.galera_sr_shutdown_master 'innodb' [ pass ]  40703
--------------------------------------------------------------------------
The servers were restarted 0 times
Spent 40.703 of 60 seconds executing testcases

Completed: All 1 tests were successful.

Reply:
This warning may or may not appear in the log. If you use option --repeat it eventually will.
On shutdown, the client thread that has SR transaction running is closed. Before closing it rolls back, and attempts to replicate a rollback fragment, this may or may not succeed during shutdown. 
But it does not really matter, during shutdown, if this replication fails.